docs/userguide/networking/get-started-overlay.md

```diff
@@ -16,7 +16,6 @@ network. Docker Engine supports multi-host networking out-of-the-box through the
 `overlay` network driver.  Unlike `bridge` networks, overlay networks require
 some pre-existing conditions before you can create one. These conditions are:
 
-* A host with a 3.16 kernel version or higher.
```

Comment:
It is going to work on all Docker Engine supported kernel versions. There are no specific minimal kernel version needs for overlay driver. Should we still specify a minimal kernel version?